CONTRIBUTING.md

```diff
@@ -99,6 +99,59 @@ This also sets up gulp and Jekyll to watch for file changes to the `/docs`
 and `/src` directories and rebuilds the website accordingly. The command is
 aliased by `npm start`
 
+### Committing files when updating the `/src` directory
+
+When you run `npm start` to preview the website locally, many files are
+generated that are tracked by Git. This leaves your working directory in a dirty
+state and knowing which files need to be included in your commit can seem like a
+daunting task. The following steps can help in committing changes while the team
+looks into different approaches to handling changes to both the Standards
+website and the `uswds` package.
+
+If you made any changes to the `/src` directory, you have also made changes in
+the following directories. This is due to how the Standards website consumes
```

Comment:
I'd add a colon after "following directories" (to indicate a list), and then move "This is due to how ..." to below the list.